doc/builders/images/appimagetools.xml

```diff
@@ -80,7 +80,7 @@ appimageTools.wrapType2 { # or wrapType1
      <varname>src</varname> specifies the AppImage file to extract.
     </para>
    </callout>
-   <callout arearefs='ex-appimageTools-wrapping-2'>
```

Comment:
It isn't a think I'm asking you to change, just something that crossed my mind. A great thing about these callouts is that they aren't inherently numbered. Concretely, these could be named `ex-appimageTools-wrapping-name`, `ex-appimageTools-wrapping-src`, `ex-appimageTools-wrapping-extraPkgs` and reduce the likelihood of mixing them up at reference time :)